qubesdb-tools/template-from-qubesdb

```diff
@@ -13,7 +13,7 @@ def get_env(args):
             value = db.read(f"/vm-config/{key}")
             if not value or len(value) == 0:
                 raise KeyError(f"Could not read from QubesDB: {key}")
-            env[key] = (value or "").decode()
+            env[key] = value.decode()
```

Comment:
@legoktm, the precedents here are:

https://github.com/freedomofpress/securedrop-client/blob/e50e0ffc2afdda318dc20aaee55c41f3e010fdcf/client/securedrop_client/config.py#L54-L56
https://github.com/freedomofpress/securedrop-client/blob/e50e0ffc2afdda318dc20aaee55c41f3e010fdcf/proxy/src/config_qubesdb.rs#L25-L29

If you can think of a better (and probably more-Pythonic) implementation of consistent behavior between the Python and Rust implementations, I'd love to standardize on that.

Reply:
I think the code is (now) the same across both Python implementations? After the `if not value or len(value) == 0:` check, `value` can assumed to be a `str` that is safe to use. Am I missing something?

Reply:
I agree they're now the same.  I can't tell if I'm getting hung up on (a) using the Rusty conditional rather than the more Pythonic try/except block or (b) having three implementations in two languages.  I don't think it's worth factoring out further; I was just wondering if you could see a more elegant implementation.  :-)

Reply:
Ah OK. I think a) is just conventions between the two languages differing. b) is not perfect, but small enough to copy-paste rather than trying to figure out how to properly do a dependency tree, etc. Maybe we can upstream the Python if/else to QubesDB? And eventually when other stuff gets oxidized, we split the proxy qubesdb code into its own crate library.